packages/seeder/src/elastic-search/schema.ts

```diff
@@ -21,6 +21,12 @@ export const productSearchSchema = {
   'info.price': {
     type: 'integer',
   },
+  'info.oldPrice': {
+    type: 'integer',
+  },
```

Comment:
I think we don't need old price in es data

Reply:
Frontend may calculate/show discounted/offer price.